src/main/java/Deadline.java

```diff
@@ -0,0 +1,18 @@
+public class Deadline extends Task{
```

Comment:
Wrong spacing for braces
```suggestion
public class Deadline extends Task {
```